conda_lock/src_parser/requirements_txt.py

```diff
@@ -0,0 +1,48 @@
+"""
+Parses pip requirements from a requirements file
+"""
+from packaging._parser import parse_requirement
+from pip._internal.network.session import PipSession
+from pip._internal.req.req_file import parse_requirements
+
+from conda_lock.models.lock_spec import VersionedDependency, VCSDependency, URLDependency, Dependency
+from conda_lock.src_parser.pyproject_toml import unpack_git_url
+
+
+def parse_requirements_txt(file_path, category=None) -> Dependency:
```

Comment:
For the return type annotation, this seems to be `Dependency` generator rather than returning a `Dependency`.